solidity/dashboard/src/sagas/liquidity-rewards.js

```diff
@@ -89,6 +89,11 @@ function* fetchLiquidityRewardsData(liquidityRewardPair, address) {
       )
     }
 
+    const rewardMultiplier = yield call(
```

Comment:
We probably should update `rewardMultiplier` when the user has withdrawn or deposited more tokens.